src/lexoRank/lexoRank.ts

```diff
@@ -311,6 +311,69 @@ export class LexoRank {
     return new LexoRank(this.bucket, LexoRank.between(this.decimal, other.decimal));
   }
 
+  public betweenLexoRanks(other: LexoRank, ranksToGenerate: number): LexoRank[] {
+
+    if (!this.bucket.equals(other.bucket)) {
+      throw new Error('Between works only within the same bucket');
+    }
+
+    if (ranksToGenerate == 0) {
+      return [];
+    }
+
+    let newLeft: LexoRank = this;
+    let newRight: LexoRank = other;
```

Comment:
I think just calling these `left` and `right` makes more sense - these are the existing lexoranks, so calling them `new` is in my opinion confusing.

Reply:
Updated variable name.